gordo_components/server/server.py

```diff
@@ -123,10 +131,55 @@ def base_healthcheck():
     return app
 
 
-def run_server(host: str = "0.0.0.0", port: int = 5555, debug: bool = False):
-    app = build_app()
-    app.run(host, port, debug=debug, threaded=False)
+def run_cmd(cmd):
```

Comment:
This function is called once, correct? Is it worth it to have an entirely different function for one line? and what I see it doesn't handle `CalledProcessError` and/or `OSError` like the doc says; it just propagates it out like it would anyway?

Reply:
I ended up doing that to make the tests straight forward for checking error cases. I need to update the doc. The way I use `check_output` now correctly handles CalledProcessErrors and OSErrors, but without extra handling on our side.